xml/obs_build_config.xml

```diff
@@ -233,6 +233,32 @@ Keep: debbuild</screen>
         </para>
        </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><parameter>obsgendiff:</parameter></term>
+       <listitem>
+        <para>
+          OBS can run a program to compare the result of the current build with
+          the previous result, e.g. to generate a difference or a changelog from
+          the diff.
+        </para>
+        <para>
+          OBS will run all scripts in
+          <filename>/usr/lib/build/obsgendiff.d/</filename> in the buildroot
+          when this flag is set (which contains the previous build and current
```

Comment:
I don't think it contains the previous build (ignoring the "oldpackages" for `build-compare`)